doc-content/enterprise-only/installation/clustering-quartz-proc.adoc

```diff
@@ -0,0 +1,103 @@
+org.quartz.jobStore.useProperties=false
+org.quartz.jobStore.dataSource=managedDS
+org.quartz.jobStore.nonManagedTXDataSource=notManagedDS
+org.quartz.jobStore.tablePrefix=QRTZ_
+org.quartz.jobStore.isClustered=true
+org.quartz.jobStore.clusterCheckinInterval = 20000
+#============================================================================
+# Configure Datasources
+#============================================================================
+org.quartz.dataSource.managedDS.jndiURL=jboss/datasources/psbpmsDS
+org.quartz.dataSource.notManagedDS.jndiURL=jboss/datasources/quartzNotManagedDS
+# Note the configured data sources that accommodate the two Quartz schemes at the very end of the file.
+----
+endif::PAM[]
+ifdef::DM[]
++
+[source, bash]
+----
+#============================================================================
+# Configure Main Scheduler Properties
+#============================================================================
+org.quartz.scheduler.instanceName = jBPMClusteredScheduler
+org.quartz.scheduler.instanceId = AUTO
+#============================================================================
+# Configure ThreadPool
+#============================================================================
+org.quartz.threadPool.class = org.quartz.simpl.SimpleThreadPool
+org.quartz.threadPool.threadCount = 5
+org.quartz.threadPool.threadPriority = 5
+#============================================================================
+# Configure JobStore
+#============================================================================
+org.quartz.jobStore.misfireThreshold = 60000
+org.quartz.jobStore.class=org.quartz.impl.jdbcjobstore.JobStoreCMT
+org.quartz.jobStore.driverDelegateClass=org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+org.quartz.jobStore.useProperties=false
+org.quartz.jobStore.dataSource=managedDS
+org.quartz.jobStore.nonManagedTXDataSource=notManagedDS
+org.quartz.jobStore.tablePrefix=QRTZ_
+org.quartz.jobStore.isClustered=true
+org.quartz.jobStore.clusterCheckinInterval = 20000
+#============================================================================
+# Configure Datasources
+#============================================================================
+org.quartz.dataSource.managedDS.jndiURL=jboss/datasources/psbrmsDS
+org.quartz.dataSource.notManagedDS.jndiURL=jboss/datasources/quartzNotManagedDS
+# Note the configured data sources that accommodate the two Quartz schemes at the very end of the file.
+----
+endif::DM[]
++
+[IMPORTANT]
+====
+The recommended interval for cluster discovery is 20 seconds and is set in the `org.quartz.jobStore.clusterCheckinInterval` of the `quartz-definition.properties` file. Depending on your set up consider the performance impact and modify the setting as necessary.
++
+The `org.quartz.jobStore.driverDelegateClass` property that defines the database dialect. If you use Oracle, set it to `org.quartz.impl.jdbcjobstore.oracle.OracleDelegate`.
```

Comment:
Stray +. Check rendered output. Missing a verb:
The `org.quartz.jobStore.driverDelegateClass` property that defines the database dialect. Probably?:
The `org.quartz.jobStore.driverDelegateClass` property defines the database dialect. 
'Oracle' is to casual. Do you mean Oracle Real Application Clusters (just guessing, but I imagine something like that).

Reply:
Corrected and removed Oracle line.